src/windows/find_tools.rs

```diff
@@ -199,6 +203,31 @@ mod impl_ {
         include: Vec<PathBuf>,
     }
 
+    fn is_amd64_emulation_supported() -> bool {
+        // In theory we should free the library handle, but since kernel32.dll is loaded in every
+        // process and we're a short-lived process, leaking the handles doesn't matter.
```

Comment:
Some users use cc-rs at runtime instead of in `build.rs`, so that assumption isn't true.

Reply:
Ok, made a wrapping type with `Drop`.